src/content/docs/kubernetes-pixie/kubernetes-integration/get-started/changes-since-v3.mdx

```diff
@@ -0,0 +1,231 @@
+---
+title: 'Kubernetes integration: what's changed in v3'
```

Comment:
I think to use apostrophe it would need to be in double quotes like so:

```
title: "Kubernetes integration: what's changed in v3"
```

Reply:
🤦🏻‍♀️ 
Thanks for catching this!